src/app-layout/visual-refresh-toolbar/toolbar/drawer-triggers.tsx

```diff
@@ -80,9 +89,16 @@ export function DrawerTriggers({
     return 0;
   };
 
-  const { visibleItems, overflowItems } = splitItems(drawers, getIndexOfOverflowItem(), activeDrawerId ?? null);
+  const indexOfOverflowItem = getIndexOfOverflowItem();
+
+  const { visibleItems, overflowItems } = splitItems(
+    [...drawers, ...globalDrawers].filter(item => !!item.trigger),
```

Comment:
I think it is 3rd time I am leaving a comment about filtering it in a different place. Seems like something really worth fixing

Reply:
fixed